frontend/src/main/pages/PersonalSchedules/PersonalSchedulesCreatePage.js

```diff
@@ -20,9 +20,14 @@ export default function PersonalSchedulesCreatePage() {
     toast(`New personalSchedule Created - id: ${personalSchedule.id} name: ${personalSchedule.name}`);
   }
 
+  const onError = (error) => {
+    // toast(`Error: ${error}`);
```

Comment:
Can we remove this commented out line of code?

Reply:
Sorry I missed this. I will fix it right now.